src/devices/Mcp23xxx/Mcp23xxx.cs

```diff
@@ -439,11 +438,48 @@ private byte GetMappedAddress(Register register, Port port = Port.PortA, BankSty
             return port == Port.PortA ? address : address += 0x10;
         }
 
+        public void OpenPin(int pinNumber) => SetPinMode(pinNumber, PinMode.Input);
+
         public void OpenPin(int pinNumber, PinMode mode) => SetPinMode(pinNumber, mode);
 
         public void ClosePin(int pinNumber)
         {
             // No-op
         }
+
+        public bool IsPinOpen(int pinNumber)
+        {
+            ValidatePin(pinNumber);
+            return true;
```

Comment:
This might be misleading right? I know that there is no such thing as opening a pin, but I'm not sure if we should keep some sort of internal state of what pins are actually being used and which ones aren't. I'm just thinking out loud here, I'm not suggesting to change this, just want to chat about it.

Reply:
Same as other comment above